### ISSUE TITLE
Show Readme, License, and Manifest

### ISSUE THREAD

**Issue by jag3773** [unfoldingWord-dev/door43.org#674]
## Story

As a *contributor* I want to post content that is out of the RC spec and still have it show on the live site so that I can share my work with others.

As a *user* I want to see basic information about a project so that I know what the project is and what I can do with it.

## Notes

The goal here is to add some Javascript to display the Readme file, License file, and manifest file to each project's page.  I suggest using a JS library for converting the markdown in the readme and license files to html (something like https://github.com/showdownjs/showdown for example).

The manifest file should also be displayed in a friendly manner, I'm not sure if there is a YAML to HTML JS library but that would be handy.

If conversion fails, the JS should still work to load the README file by default.

Work with @jag3773 on the design of the page.

Test case: https://git.door43.org/jaxmark/NTGreekoutlines

**Comment by RobH123** [comment]
Other test cases:

https://git.door43.org/BCS-BIBLE/Hindi_UDB_OT
https://git.door43.org/WycliffeAssociates/en_map
https://git.door43.org/unfoldingWord/en_obs-sn
ugl? ugc?

**Comment by RobH123** [comment]
@jag3773 Thinking about this, I guess doing it in JS (at page view time) is one way. I think it could also be done in Python in the tX system (like the way RC compatible repos are done).

Seems that either way would work?

**Comment by jag3773** [comment]
@RobH123 Correct, either way would work.  Doing it in JS has the advantage of working for all repos without having to trigger a rebuild of all of the repositories.  I think it has the merit of being more flexible in the long term too.

**Comment by RobH123** [comment]
@jag3773 Ok, it was so easy to pass the ReadMe in the tX system (if the preprocessor don't find other files), that I went ahead and did that anyway.

This doesn't obviate the need for a JS solution which would work without needing to trigger rebuilds as you correctly say above.

**Comment by RobH123** [comment]
@jag3773 Ok, should I still try to work on a JS version? Or what will we do about the new Preview tab, i.e., will that obsolete this issue?

**Comment by jag3773** [comment]
I think we can close this out for now, we can revisit later if we decide to do something different with the Preview tab.

**Comment by RobH123** [comment]
@jag3773 I wonder if I misunderstood this issue, or maybe the issue changed over time. (I think at first it was for sites that didn't convert successfully.) Anyway, while working on content, I've realised that the ReadMe's contain important information for users (not just for content creators), yet it's not currently displayed on pages like https://door43.org/u/unfoldingWord/en_ult/. Do we need to reopen/redefine this issue?

**Comment by RobH123** [comment]
Update: After some research, I think uw Bible texts need front matter pages and this would be a better solution than rendering the ReadMe files with tX,

**Comment by jag3773** [comment]
Yeah, I think that would be good @RobH123 , I would like to see those files on all projects as the original user stories indicate.
